tests/test_internals/test_interfaces.py

```diff
@@ -59,9 +59,8 @@ class TutorialNWBConverter(NWBConverter):
     metadata["Subject"] = dict(subject_id="Name of imaginary testing subject (required for DANDI upload)")
     conversion_options = dict(RecordingTutorial=dict(stub_test=stub_test), SortingTutorial=dict())
     converter.run_conversion(
-        metadata=metadata,
         nwbfile_path=output_file,
```

Comment:
Maybe we can the `output_file` definition (line 49) to nwbfile_path so it coincides with the naming here?

Reply:
Wow lol that takes me back... fixed